pkg/worker/translate_audit_job.go

```diff
@@ -0,0 +1,55 @@
+package worker
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	faktory_worker "github.com/contribsys/faktory_worker_go"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/mint-app/pkg/translatedaudit"
+)
+
+// TranslateAuditJob is the job to translate an individual audit into an analyzed audit, and note that the translation is done in the processing table
+// args[0] the id of the audit change (int, but float in faktory)
+// args[0] the id of the queue (UUID)
+func (w *Worker) TranslateAuditJob(ctx context.Context, args ...interface{}) (returnedError error) {
+	/*
+		// Future Enhancement: the job is wrapped in panic protection when it is registered, BUT it won't update the queue on a panic. If desired we can also defer the panic here, and try to enable the recover on the parent function to update the queue item
+		// defer apperrors.RecoverPanicAsErrorFunction(&returnedError)
+		// fmt.Printf("translating audit job reached. Args %v", args)
```

Comment:
Mark with //TODO: ... for IDE capture

Reply:
// Future Enhancement: can be captured with IDE, (I for example use [TODO-TREE](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree)). There are other instances of this in the codebase. I'd like to leave this as is.